documentation/modules/retrieving-vmware-mor.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: PROCEDURE
+[id="retrieving-vmware-mor_{context}"]
+= Retrieving a VMware vSphere MOR
+
+When you migrate VMs with a VMware vSphere source provider using {project-first}, you need to know the managed object reference (MOR) of the provider. You retrieve the MOR using a procedure that is very similar to the procedure for xref:retrieving-validation-service-json_{context}[Retrieving the Inventory service JSON]. For your convenience, the procedure that follows contains the steps the procedures have in common as well the ones that are unique for retrieving the MOR.
+
+
+.Procedure
+
+. Retrieve the routes for the project:
++
+[source,terminal]
+----
+oc get route -n openshift-mtv
+----
+
+. Retrieve the `Inventory` service route:
++
+[source,terminal,subs="attributes+"]
+----
+$ {oc} get route <inventory_service> -n {namespace}
+----
+
+. Retrieve the access token:
++
+[source,terminal]
+----
+$ TOKEN=$(oc whoami -t)
+----
+
+. Trigger an HTTP GET request (for example, using Curl):
++
+[source,terminal]
+----
+$ curl -H "Authorization: Bearer $TOKEN" https://<inventory_service_route>/providers -k
+----
+
+. Retrieve the `MOR` of a VMware vSphere provider:
++
+[source,terminal]
+----
+$ curl -H "Authorization: Bearer $TOKEN"  https://<inventory_service_route>/providers/<vsphere> -k
+----
+
+. Retrieve the datastore of a VMware vSphere source provider:
++
+[source,terminal]
+----
+$ curl -H "Authorization: Bearer $TOKEN"  https://<inventory_service_route>/providers/<datastore>/<MOR>/vms -k
```

Comment:
but when we get here, we don't know the MOR yet so we should rather ask the user to list the datastores:
```suggestion
$ curl -H "Authorization: Bearer $TOKEN"  https://<inventory_service_route>/providers/vsphere/<provider id>/datastores/ -k
```
and then find the datastore in the output, for example:
```json
[
  {
    "id": "datastore-11",
    "parent": {
      "kind": "Folder",
      "id": "group-s5"
    },
    "path": "/Datacenter/datastore/v2v_general_porpuse_ISCSI_DC",
    "revision": 46,
    "name": "v2v_general_porpuse_ISCSI_DC",
    "selfLink": "providers/vsphere/01278af6-e1e4-4799-b01b-d5ccc8dd0201/datastores/datastore-11"
  },
  {
    "id": "datastore-730",
    "parent": {
      "kind": "Folder",
      "id": "group-s5"
    },
    "path": "/Datacenter/datastore/f01-h27-640-SSD_2",
    "revision": 46,
    "name": "f01-h27-640-SSD_2",
    "selfLink": "providers/vsphere/01278af6-e1e4-4799-b01b-d5ccc8dd0201/datastores/datastore-730"
  },
 ....
```
and the `id` field would indicate the MOR of the datastore